### DISCUSSION TITLE
Medicod

### DISCUSSION THREAD

**Discussion by PRANJALRANA11** [OpenCodeEra/.github#24] [Ideas]
MEDICOD, where your health information is our top priority. We offer a secure and user-friendly platform to store and manage your medical records online
https://github.com/PRANJALRANA11/MEDICOD

**Comment by AdarshRawat1** [comment]
@PRANJALRANA11 How do you plan to include this repository in our organization?
- By transferring ownership of repository .
- By creating a Fork of your repository .
- By create a new repo and hosting the code in it.

**Comment by PRANJALRANA11** [comment]
@AdarshRawat1  I want to make a repo and host my code in it

**Comment by AdarshRawat1** [comment]
@PRANJALRANA11 I have sent you organization's invitation. Accept it and create a new repository for your project. 

Looking forward to a great project!!

**Comment by PRANJALRANA11** [comment]
@AdarshRawat1 Done it
